<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyScanClass</class>
 <widget class="QMainWindow" name="KeyScanClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeyScan</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <widget class="QWidget" name="NavbarContainer" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: gray;
    padding: 5px;
    border-radius: 4px;
    font-weight: bold;
    margin: 7px; 
}

QPushButton:hover {
    background-color: white;
    border: 1px solid #CCCCCC
}
</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>105</width>
       <height>691</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="NavbarLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="home_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoHome"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="keyboard_tester_button">
        <property name="text">
         <string>KeyTest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="typing_test_button">
        <property name="text">
         <string>TypingTest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="key_remapper_button">
        <property name="text">
         <string>KeyRemapper</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="KeyboardContainer" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>951</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: #E0E0E0;
        border: 1px solid #A0A0A0;
        border-radius: 6px;
		padding: 5px 10px;
        font: bold 12px &quot;Arial&quot;;
        min-width: 20px;
		margin: 2px;
        min-height: 20px;;
    }

    QPushButton:hover {
        background-color: #D0D0D0;
    }

    QPushButton:pressed {
        background-color: #C0C0C0;
        border-style: inset;
        padding-top: 6px;
    }

    QPushButton:disabled {
        background-color: #F0F0F0;
        color: #A0A0A0;
    }</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="row5_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="row4_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="row2_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="f1_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="row3_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="row6_layout"/>
    </widget>
    <widget class="QWidget" name="KeyArrowsContainer" native="true">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>269</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="bottom_arrow_layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="top_arrow_layout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>30</y>
       <width>160</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="row_7_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>90</y>
       <width>160</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="row_8_layout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KeyScan.qrc"/>
 </resources>
 <connections/>
</ui>
